src/components/Filter/Filter.style.ts

```diff
@@ -43,25 +44,37 @@ export const buttonWrapperStyle = ({
     height: '100%',
     minWidth: rem(110),
 
-    ':hover > div, :active > div': {
+    ':hover > div': {
       backgroundColor:
         !disabled && !open
           ? hasSelectedValue
-            ? theme.utils.getColor(calculatedColor.color, 100)
-            : theme.utils.getColor('darkGrey', null, 'pale')
+            ? getHover({ theme, color: calculatedColor.color, shade: calculatedColor.shade })
+                .backgroundColor
+            : getHover({ theme }).backgroundColor
+          : undefined,
+    },
+
+    ':active > div': {
```

Comment:
this is difficult to capture can we have a section with comments explain the logic ?